docs/contributing/semgrep-core-contributing.md

```diff
@@ -169,6 +173,9 @@ You can also instead set the environment variable SEMGREP_CORE_PROFILE to 1 to g
 cd semgrep-core
 export SEMGREP_CORE_PROFILE=1
 ./bin/semgrep-core -e foo tests/python
+```
+will output:
+```bash
 ---------------------
 profiling result
 ---------------------
```

Comment:
Found another code snippet where this confusion was not caused by me removing the $-sign.

Reply:
Thank you!